### ISSUE TITLE
Internet access.

### ISSUE THREAD

**Issue by Weaver2World** [nomadbsd/NomadBSD#61]
On following the onsite instructions the `fetch https' instruction fails, because `fetch' is an unrecognised command in Debian.
On navigating to _/config/etc/rc.d/_, I find that doesn't appear to exist to facilitate the download.
All other aspects of the install went well, except for the Internet access aspect.
Where am I going wrong?
Thanks for any time and trouble taken with this.

**Comment by mrclksr** [comment]
Hi @Weaver2World ,

I'm not quite sure I understand. You tried to build (running the build script) a NomadBSD image on a Debian system? If that's the case, this won't work. You need a FreeBSD system which has a ABI compatible with the base system of the NomadBSD image you intend to build.

**Comment by Weaver2World** [comment]
No, I already have Nomad installed on a flash drive, but have no Internet access through it.
On going to the Git site to get the required file (setup_netifs), as I can't download it, I find it doesn't virtually exist.

https://nomadbsd.org/download/errata-1.2.txt

It's quite possible to install Nomad from a Debian system, because I've done it from my Debian \home folder to the thumb drive on an ext4 format. It's up and running. All I lack is Internet access from it.

**Comment by bsdlme** [comment]
The setup_netifs script is only needed for the 1.2 version of NomadBSD. If you download and boot 1.3.1 the script is already included.
